### ISSUE TITLE
Answer this: Settings FINAL

### ISSUE THREAD

**Issue by corinnewo** [ji-devs/ji-cloud#3291]
- [x] Take out the finishing options "Student finishes this activity by..."
- [x] For option "Would you like questions to be asked in order?", the default should be 'ask in order', and it should be on the left.
- [x] Put the timer as the last option
- [x] Add in option "Highlight correct answer..." and have options "after tries" (pulldown says "after [x] tries", default is 3 tries)  and "don't highlight"
- [x] Timer:
  - [x] Change text for timer question to make it clear that it is the time PER QUESTION and not per activity: Change "Would you like to set a time limit?" to "Would you like to set a time limit per question?" 
  - [x] the second option has the per question addition. 
  - [x] Timer restarts at the start of each question. Default is 10 seconds.
  - [x] Stop timer when a question is completed correctly.
- [x] If there are multiple questions, then the NEXT in the player should go to the next question and not the next module.
  - [x] If the student is on the last question, the next button goes to the next module; If they're on the first question, the back button goes to the previous module.
- [x] If there is a highlight after x tries, then briefly blink the answer and let it disappear.

@GaliShapira please add in link to Zeplin

**Comment by GaliShapira** [comment]
Zeplin:
https://zpl.io/dxjnxnE

Screenshot:
<img width="468" alt="Screen Shot 2022-10-25 at 15 37 13" src="https://user-images.githubusercontent.com/88834002/197774839-a74806ce-bbe3-4e57-b163-ed06591301e4.png">

**Comment by chayaml** [comment]
@corinnewo 
 What if we changed the default from "clicking on continue" to "clicking all items''? I always go to change that default and I think other creators would prefer it too (having all items they included completed)... what do you think?
![Screen Shot 2022-11-13 at 8 54 21 PM](https://user-images.githubusercontent.com/78186896/201560439-d0b23126-e389-47cc-a97e-979c7e8fdc3f.png)

**Comment by corinnewo** [comment]
@chayaml We don't have this setting here in Answer this. In other settings we have also taken out the 'minimum' setting, as we felt that the clicking continue would have the same effect. Look at what we did here: https://github.com/ji-devs/ji-cloud/issues/3286. The default is clicking next, as we felt that in listen and learn, we did not want to force them to listen to ALL the options. Happy to discuss!
